packages/experimental-loader/src/warning.ts

```diff
@@ -18,3 +17,39 @@ export class Warning extends Error {
         this.stack = undefined;
     }
 }
+
+interface PostCSSError {
+    reason: string;
+    line: string;
+    column: string;
+    showSourceCode(): string;
+}
+
+export class CssSyntaxError extends Error {
+    constructor(error: PostCSSError) {
+        super(error.toString());
+
```

Comment:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-2.html#support-for-newtarget